static/js/settings_bots.js

```diff
@@ -56,6 +56,14 @@ exports.generate_zuliprc_content = function (email, api_key) {
            "\n";
 };
 
+exports.generate_zuliprc_content_for_single_file = function (email, api_key) {
+    return "[" + email.substring(0, email.indexOf("-bot@zulip.localhost")) + "]" +
```

Comment:
Will this always be `@zulip.localhost`?  Would it make sense to use `email.indexOf("-bot@")` instead?